GI01SUMO/GI01SUMO_R00001_ingest.csv

```diff
@@ -1,80 +1,83 @@
-mi.dataset.driver.nutnr_b.dcl_conc.nutnr_b_dcl_conc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl11/NUTNR*/*.nutnr1.log,GI01SUMO-SBD11-08-NUTNRB000,recovered_host,Available,
-#mi.dataset.driver.nutnr_b.nutnr_b_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl11/NUTNR*/*.DAT,GI01SUMO-SBD11-08-NUTNRB000,recovered_inst,Not Expected,no inst data available
-mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/syslog*/*syslog*.log,GI01SUMO-SBD12-00-DCLENG000,recovered_host,Available,
-mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/OPTAA*/*.optaa*.log,GI01SUMO-SBD12-01-OPTAAD000,recovered_host,Available,
-mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/FLORT*/*.flort1.log,GI01SUMO-SBD12-02-FLORTD000,recovered_host,Available,
-mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/HYDGN_sn/*hyd*.log,GI01SUMO-SBD12-03-HYDGN0000,recovered_host,Available,
-mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/PCO2A*/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
-mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/WAVSS*/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
-mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/METBK-Log_snLGR005/*.metbk*.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
-mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl12/FDCHP*/*.fdchp.log,GI01SUMO-SBD12-08-FDCHPA000,recovered_host,Available,
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl16/syslog*/*syslog*.log,GI01SUMO-RID16-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl16/OPTAA*/*.optaa*.log,GI01SUMO-RID16-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl16/FLORT*/*.flort2.log,GI01SUMO-RID16-02-FLORTD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl16/CTDBP*/*ctdbp.log,GI01SUMO-RID16-03-CTDBPF000,recovered_host,Available,
+#mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00001/cg_data/dcl16/CTDBP*/*.hex,GI01SUMO-RID16-03-CTDBPF000,recovered_inst,Not Expected,no inst data available due to damaged Y-cable

```

Comment:
A damaged Y-Cable just means we could get the data at sea. But we should have downloaded it back on shore, but I don't see it in the Raw Data Repo.